### PR TITLE
ODH Operator OLM release v1.1.1

### PR DESCRIPTION
**Pull request by LaVLaS**
OLM  catalog files update for v1.1.1

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/1.1.0/manifests/opendatahub-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -267,75 +275,87 @@ spec:
...
-    url: https://www.kubeflow.org/
-  - name: Kubeflow on OpenShift
-    url: https://www.kubeflow.org/docs/openshift/
+    - name: Open Data Hub
+      url: https://opendatahub.io
+    - name: Open Data Hub Community
+      url: https://github.com/opendatahub-io
+    - name: Kubeflow
+      url: https://www.kubeflow.org/
+    - name: Kubeflow on OpenShift
```
[the remark this answers is not in the document]
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.1.0/manifests/opendatahub-operator.v1.1.0.clusterserviceversion.yaml:357]: Which kubeflow `github.com` link?

[the remark this answers is not in the document]
Comment (nakfour) [reply deploy/olm-catalog/opendatahub/1.1.0/manifests/opendatahub-operator.v1.1.0.clusterserviceversion.yaml:357]: Let leave it for now, we will update it in the next release

**Comment by LaVLaS** [comment]
https://github.com/openshift/release/pull/23307 should fix the CI failures.  The CI jobs are trying to install from the `beta` channel that was deprecated and replaced with `rolling` for the latest ODH releases

**Pull request merged**